src/processes/follow-topic-process.js

```diff
@@ -92,9 +94,49 @@ const sendMessageAsAnonymous = async (serverClient, communityMessageFormat, data
   }
   // }
 };
+
+const sendSignedMessage = async ({
+  serverChannel,
+  targetUserId,
+  senderUsername,
+  communityMessageFormatUserId,
+  text
+}) => {
+  const message = await serverChannel?.sendMessage(
+    {
+      text,
+      user_id: communityMessageFormatUserId,
+      is_auto_message: true
+    },
+    {skip_push: true}
+  );
+
+  const notificationPayload = {
+    title: senderUsername,
+    body: `${text?.substring(0, 100)}`
+  };
+
+  const dataPayload = {
+    type: 'message.new',
+    channel_id: serverChannel?.id,
+    message: text,
+    message_schema: 'text',
+    attachment: '',
+    created_at: message?.message?.created_at || new Date().toISOString(),
+    is_annoymous: 'false',
+    priority: 'high',
+    content_available: 'true'
+  };
+
+  sendChatNotificationByChannelMembers([targetUserId], {
+    notification: notificationPayload,
+    data: dataPayload
+  });
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The newly introduced function `sendSignedMessage` seems to be well implemented. It sends a message using the server channel and then prepares a notification payload and data payload which are used to send a chat notification by channel members. However, there's no error handling in case `sendChatNotificationByChannelMembers` fails. Consider adding a try-catch block around it or ensure that `sendChatNotificationByChannelMembers` handles its own errors gracefully.
```diff
  sendChatNotificationByChannelMembers([targetUserId], {
    notification: notificationPayload,
    data: dataPayload
  });
+ // Add error handling here
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->